src/python/evaluation/common/xlsx_util.py

```diff
@@ -0,0 +1,43 @@
+import logging.config
+from pathlib import Path
+from typing import NoReturn, TypeVar, Union
+
+import pandas as pd
+from openpyxl import load_workbook, Workbook
+from src.python.evaluation.evaluation_config import EvaluationConfig
+
+logger = logging.getLogger(__name__)
+E = TypeVar('E', bound=EvaluationConfig)
+
+
+def remove_sheet(workbook_path: Union[str, Path], sheet_name: str, to_raise_error=False) -> NoReturn:
```

Comment:
```suggestion
def remove_sheet(workbook_path: Union[str, Path], sheet_name: str, to_raise_error=False) -> None:
```
> The typing module provides a special type NoReturn to annotate functions that never return normally
https://www.python.org/dev/peps/pep-0484/#the-noreturn-type